src/core-services/catalog/queries/tags.js

```diff
@@ -12,19 +12,44 @@ import _ from "lodash";
  * @param {Boolean} [params.isTopLevel] - If set, look for `isTopLevel` matching this value
  * @param {Boolean} [params.shouldIncludeDeleted] - Admin only. Whether or not to include `isDeleted=true` tags. Default is `false`
  * @param {Boolean} [params.shouldIncludeInvisible] - Admin only. Whether or not to include `isVisible=false` tags.  Default is `false`.
+ * @param {Boolean} [params.excludedTagIds] - If set, exclude these tagIds from the result
  * @returns {Promise<MongoCursor>} - A MongoDB cursor for the proper query
  */
-export default async function tags(context, shopId, { filter, shouldIncludeDeleted = false, isTopLevel, shouldIncludeInvisible = false } = {}) {
+export default async function tags(
+  context,
+  shopId,
+  {
+    filter,
+    shouldIncludeDeleted = false,
+    isTopLevel,
+    shouldIncludeInvisible = false,
+    excludedTagIds
```

Comment:
This needs to be added as a field to GQL schema here for the tags query:
`/src/core-services/tags/schemas/tags.graphql`

I'd assume it would be of the following type

```graphql
excludedTagIds: [ID]
```

Then the ids would need to be decoded in this tags query resolver:
`src/core-services/tags/resolvers/Query/tags.js`.

Heres a diff of those changes some changes I made locally which seem to work.

```diff
diff --git a/src/core-services/tags/resolvers/Query/tags.js b/src/core-services/tags/resolvers/Query/tags.js
index 01778b677..0b8049dc6 100644
--- a/src/core-services/tags/resolvers/Query/tags.js
+++ b/src/core-services/tags/resolvers/Query/tags.js
@@ -1,6 +1,6 @@
 import getPaginatedResponse from "@reactioncommerce/api-utils/graphql/getPaginatedResponse.js";
 import wasFieldRequested from "@reactioncommerce/api-utils/graphql/wasFieldRequested.js";
-import { decodeShopOpaqueId } from "../../xforms/id.js";
+import { decodeShopOpaqueId, decodeTagOpaqueId } from "../../xforms/id.js";
 
 /**
  * Arguments passed by the client for a tags query
@@ -26,11 +26,19 @@ import { decodeShopOpaqueId } from "../../xforms/id.js";
  * @returns {Promise<Object[]>} Promise that resolves with array of Tag objects
  */
 export default async function tags(_, connectionArgs, context, info) {
-  const { shopId } = connectionArgs;
+  const { shopId, excludedTagIds } = connectionArgs;
 
   const dbShopId = decodeShopOpaqueId(shopId);
+  let dbExcludedTagIds;
 
-  const query = await context.queries.tags(context, dbShopId, connectionArgs);
+  if (Array.isArray(excludedTagIds)) {
+    dbExcludedTagIds = excludedTagIds.map(decodeTagOpaqueId);
+  }
+
+  const query = await context.queries.tags(context, dbShopId, {
+    ...connectionArgs,
+    excludedTagIds: dbExcludedTagIds
+  });
 
   return getPaginatedResponse(query, connectionArgs, {
     includeHasNextPage: wasFieldRequested("pageInfo.hasNextPage", info),
diff --git a/src/core-services/tags/schemas/tags.graphql b/src/core-services/tags/schemas/tags.graphql
index 5c060e7d4..53ea18df6 100644
--- a/src/core-services/tags/schemas/tags.graphql
+++ b/src/core-services/tags/schemas/tags.graphql
@@ -111,6 +111,9 @@ extend type Query {
     "If provided, this query will do a regex search using the provided filter data, and return only tags that match"
     filter: String
 
+    "Tags to exclude from results"
+    excludedTagIds: [ID]
+
     "If set, the query will return only top-level tags or only non-top-level tags. By default, both types of tags are returned."
     isTopLevel: Boolean,
 ```